benchmark/slate/changes/normalize.js

```diff
@@ -35,5 +36,5 @@ const value = (
 )
 
 module.exports.input = function() {
-  return value.change()
+  return new Editor({ value }, { normalize: false })
```

Comment:
what's the lifetime of this normalize: false flag? does it expire when the value is set to the editor?

Reply:
It is only used in the initial `setValue` call. I figured it might defeat the purpose of the benchmark if we were already per-normalizing the value before the initial run.

https://github.com/ianstormtaylor/slate/blob/f677f7655928ee6a848f997e4f3361d4fb1b5705/packages/slate/src/controllers/editor.js#L67